src/main/java/frc/robot/commands/CmdManipulatorSetGraspPosition.java

```diff
@@ -17,6 +18,7 @@ public CmdManipulatorSetGraspPosition(double graspPosition)
     @Override
     public void initialize() 
     {
+        RobotLog.getInstance().log("Setting grasp to " + _graspPosition);
```

Comment:
Since we want the log to be readable, let's format our output a little nicer.

I think you're logging information correctly, but the issue we'll run into is that we'll end up with a line that looks something like:

```
Setting grasp to 12.47520000001
```

To resolve this, you'll need to use the `String.format` method, which will allow you to round the position to a specific number of decimal places:

```suggestion
        RobotLog.getInstance().log(String.format("Setting grasp to %6.2f", _graspPosition));
```